### ISSUE TITLE
Implement functionality on view video page

### ISSUE THREAD

**Issue by arthurmcgregor** [TheCacophonyProject/cacophony-web-vuex#2]
- [x] Display devicename and date/time
- [x] Display and auto-play video
- [x] Populate tag table
  - [x] Fix animal image not showing
  - [ ] Add 'Me!' if tag added by user
  - [x] Add background colour to AI IDs
  - [x] Activate delete button
- [x] Quick tag buttons
  - [x] Add new tag when animal button clicked
  - [x] Add false positive event when fp button clicked
  - [x] Display AddObservation component when + clicked
- [x] Prev/next buttons
  - [x] Bug - prev*/next* buttons returning server error
- [x] Processing status
- [x] Display comment
- [x] Update comment
- [x] Download buttons
- [x] Delete button
- [x] Add animal observation
  - [x] Set time start / end / current buttons
- [ ] Set tag stop time to duration - 10s (??)
- [ ] Confirm responsive / mobile views
- [ ] Handle no recording at URL

**Comment by mjs** [comment]
Nice work!

**Comment by arthurmcgregor** [comment]
@mjs I think this can probably be closed, at least for v1 release. With regard to the outstanding items:
- Add 'Me' if tag added by users -> nice to have but not essential?
- Set tag stop time to duration - 10s -> I never knew why this was in the original code!! Maybe not needed?
- Mobile view -> might not be the tidiest but I think it's functional
- Handle no recording -> an error is displayed as part of global messaging... not sure how else to handle it except perhaps redirect back to search page?

**Comment by mjs** [comment]
@arthurmcgregor 

* Agreed that adding "me" for new tags is a nice to have. I've added #68 to track this.
* The tag stop time thing was just a convenience in case you wanted to tag a particular part of the recording. 10s from the end is often when the interesting stuff stopped happening. Tagging is going to change quite a bit soon so we probably don't need this.
* I've just tried it out on my mobile and although it's not perfect it's actually not too bad. Agreed that this can wait.
* The current no recording handling is fine.

Closing this issue!
